english/see-smore-options/see-smore-options.md

```diff
@@ -0,0 +1,6 @@
+Not sure of what to do, you decide to phone-a-friend for help.  You hit speed dial for your friend, Graham Hershey.  Graham picks up after three rings, but you sense this is not going to end well.  Graham sounds like he's already toasted, and the background noise suggests he's sitting in a bar.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.